CHANGELOG.md

```diff
@@ -1,8 +1,10 @@
 # Changelog
 
-## [0.2.4] -- 2024-06-26
+
+## [0.2.5] -- 2024-06-26
```

Comment:
This is a little weird, did you mean to drop the 0.2.4 section?

Reply:
Yeah, since I'm going to re-release this I need to raise the version

Reply:
Ok. Just wasn't sure if it was as you intended.